pkg/flowkit/arguments.go

```diff
@@ -114,3 +120,53 @@ func ParseArguments(args []string, argsJSON string) (scriptArgs []cadence.Value,
 
 	return
 }
+
+func ParseArgumentsWithoutType(args []string, program *ast.Program, checker *sema.Checker) (scriptArgs []cadence.Value, err error) {
```

Comment:
It would be great if there was a simple test for this, so we don't lower the overall coverage (which is bad as it is). If you don't like doing it, I can give you my help.